components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -281,6 +288,188 @@ private void handleRequest(HttpServletRequest req, HttpServletResponse resp, boo
         }
     }
 
+    private void handleInvalidRequestMessage(HttpServletRequest req, HttpServletResponse resp, String sessionId)
+            throws IOException, IdentityException, ServletException {
+
+        log.debug("Invalid request message or single logout message ");
```

Comment:
Better to enclose with log.isDebugEnabled() check.
Make the log message a sentence.
Add context information.

Reply:
Fixed with 3bbc008dd576ca0169fde3b49fe4f505a99200ea